### PR TITLE
virtuoso 7.2.0

### PR DESCRIPTION
**Pull request by dunn**
Last week @DomT4 said there was a moratorium on new `desc`s; is that still in effect or should I go ahead and add one?

### CODE REVIEW

**Comment by bfontaine** [comment]
No there’s still one on existing formulae.

File: Library/Formula/virtuoso.rb [hunk 1]
```diff
@@ -1,12 +1,11 @@
-require 'formula'
-
 class Virtuoso < Formula
-  homepage 'http://virtuoso.openlinksw.com/wiki/main/'
-  url "https://downloads.sourceforge.net/project/virtuoso/virtuoso/7.1.0/virtuoso-opensource-7.1.0.tar.gz"
-  sha1 "255d275d810bdb7cfa55fef46517724823c4c561"
+  homepage "http://virtuoso.openlinksw.com/wiki/main/"
+  url "https://github.com/openlink/virtuoso-opensource/releases/download/v7.2.0.1/virtuoso-opensource-7.2.0_p1.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/virtuoso/virtuoso/7.2.0/virtuoso-opensource-7.2.0_p1.tar.gz"
+  sha256 "fe0fc64c169fc59322142d4e4a90d1328f7063a07f9d9035aaf5adc9dde28302"
```
Comment (bfontaine) [inline Library/Formula/virtuoso.rb]: Could you please add a `version "7.2.0-1"`? Homebrew fails to get the correct version here.

Comment (dunn) [inline Library/Formula/virtuoso.rb]: Ah, forgot about that, sorry.

Comment (dunn) [inline Library/Formula/virtuoso.rb]: Added below.

**Comment by bfontaine** [comment]
Thanks @dunn!

**Pull request closed without merging**